servers/src/main/java/tachyon/worker/block/BlockWorker.java

```diff
@@ -96,6 +96,8 @@
   private final UIWebServer mWebServer;
   /** Worker metrics system */
   private MetricsSystem mWorkerMetricsSystem;
+  /** Shared executor service */
+  private ExecutorService mSharedExecutor;
```

Comment:
Are the consumers of this `BlockMasterSync` and `AsyncEvictor`? Maybe this can just be called `mBlockManagementExecutor`